src/DateInput/MonthSelect.jsx

```diff
@@ -43,7 +44,7 @@ export default function MonthSelect({
     >
       {!value && (
         <option value="">
-          --
+          {placeholder !== undefined ? placeholder : '--'}
```

Comment:
Input uses `defaultProps` to handle that. Would you mind changing it so that here's just:
```suggestion
          {placeholder}
```

and at the bottom there's:

```js

MonthSelect.defaultProps = {
  placeholder: '--',
};
```
?

Reply:
Just made the update. Thanks for the quick review!